src/net_processing.cpp

```diff
@@ -914,6 +929,28 @@ void PeerManagerImpl::PushNodeVersion(CNode& pnode, int64_t nTime)
     }
 }
 
+size_t PeerManagerImpl::GetFloodingOutboundsCount(const CNode& skip_node) const
+{
+    size_t result = 0;
+    m_connman.ForEachNode([this, &result, &skip_node](const CNode* pnode) {
+        if (!pnode->m_tx_relay) return;
+        if (pnode->GetId() == skip_node.GetId()) return;
+        if (!pnode->IsFullOutboundConn() && !pnode->IsManualConn()) return;
```

Comment:
I find (!A && !B) to be very confusing. Again, would be good to extract into its own function with switch statement.